.circleci/config.yml

```diff
@@ -88,7 +88,7 @@ jobs:
       - run: docker kill portforward || exit 0
       - run: docker rm portforward || exit 0
       - run: /go/portforward.sh start
-      - run: kind create cluster --name=integration
+      - run: KIND_CLUSTER_NAME=integration kind-with-registry.sh && export KUBECONFIG="$(kind get kubeconfig-path --name="integration")"
```

Comment:
hmmm...I think what you need to do here is to split kind-with-registry.sh into two scripts: the bit that creates the cluster, and the bit that initializes the cluster.

Right now, you need to do a portforward.sh to the apiserver port (see a couple lines down) to be able to talk to the apiserver. It's failing because the kind-with-registry script tries to talk to the apiserver before that port-forward is set up

Reply:
urgh you're right. will do